api/src/pcapi/core/offers/validation.py

```diff
@@ -137,6 +137,25 @@ def check_stock_price(price: decimal.Decimal, offer: models.Offer, error_key: st
         )
         raise errors
 
+    if FeatureToggle.WIP_ENABLE_OFFER_PRICE_LIMITATION.is_active():
+        offer_price_limitation_rule = models.OfferPriceLimitationRule.query.filter(
+            models.OfferPriceLimitationRule.subcategoryId == offer.subcategoryId
+        ).one_or_none()
+        if offer_price_limitation_rule and offer.stocks:
```

Comment:
Question : tu vérifies `offer.stocks`, mais peut-on avoir simplement supprimé le stock qui a été validé, donc avoir `offer.stocks` vide tout en ayant un `offer.lastValidationPrice` ? Si oui, c'est dommage de ne pas rentrer dans le `if`.
Proposition : 
```suggestion
        if offer_price_limitation_rule and (offer.lastValidationPrice is not None or offer.stocks):
```
Cette proposition a aussi l'avantage d'éviter la requête sur les stocks (s'ils ne sont pas préchargés) lorsqu'on n'en a pas besoin.